### PR TITLE
Increase the limit of open files on Hosted Ubuntu 1604 to 50000

### PR DESCRIPTION
**Pull request by pasangh**
Increase the limit of open files on Hosted Ubuntu 1604. Reference - https://askubuntu.com/questions/162229/how-do-i-increase-the-open-files-limit-for-a-non-root-user

### CODE REVIEW

**Comment by pasangh** [comment]
@alepauly  yes. I just need to test it before merging, I'll do it next week

File: images/linux/ubuntu1604.json [hunk 1]
```diff
@@ -72,6 +72,8 @@
                 "systemctl disable apt-daily.timer",
                 "systemctl disable apt-daily-upgrade.timer",
                 "systemctl disable apt-daily-upgrade.service"
+                "echo -e '* soft nofile 50000 \n* hard nofile 50000' >> /etc/security/limits.conf",
+                "echo -e 'session required pam_limits.so' >> /etc/pam.d/common-session"
```
[the remark this answers is not in the document]
Comment (pasangh) [reply images/linux/ubuntu1604.json]: https://medium.com/@muhammadtriwibowo/set-permanently-ulimit-n-open-files-in-ubuntu-4d61064429a

[the remark this answers is not in the document]
Comment (pasangh) [reply images/linux/ubuntu1604.json]: Had tried that out locally

[the remark this answers is not in the document]
Comment (pasangh) [reply images/linux/ubuntu1604.json]: https://askubuntu.com/questions/162229/how-do-i-increase-the-open-files-limit-for-a-non-root-user this too

File: images/linux/ubuntu1604.json [hunk 2]
```diff
@@ -95,123 +97,123 @@
...
-            "execute_command": "sudo sh -c '{{ .Vars }} {{ .Path }}'"
-        },
-        {
-            "type": "shell",
-            "inline": [
-                "sleep 30",
-                "/usr/sbin/waagent -force -deprovision+user && export HISTSIZE=0 && sync"
-            ],
+                },	
+        {
```
Comment (alepauly) [inline images/linux/ubuntu1604.json]: please undo these whitespace changes, they cause unnecessary noise.

**Comment by pasangh** [comment]
Testing this changes using packer

**Comment by vladhor** [comment]
Since systemd ignores limits configured in places such as limits.conf, limits.d, sysctl etc. you have to make that change specifically in your daemon systemd file:

[Service]
LimitNOFILE=65536

**Pull request merged**